modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,7 +10,7 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate the link:https://github.com/spadgett/console-plugin-template/blob/main/template.yaml[template] by running the following command:
+. To deploy your plug-in to a cluster, instantiate the link:https://github.com/openshift/console-plugin-template/tree/release-4.10[template]your plug-in by running the following command:
```

Comment:
@sferich888 could I get special permission to add this GH link to the docs? It will also go in with #49996 as well. It is a in the openshift org and QE reviewed. It would not be for docs, just for the template. This template is used as a base for this feature, and it would be difficult for the user to get started otherwise. cc. @spadgett

Reply:
Hey, @opayne1. This isn't the link we need... You need to be able to create your GitHub repo from the openshift/console-plugin-template as a starting point for your plugin. Somewhere along the way, this was removed from the docs. As written, users won't be able to create plugins now using these docs.